src/main/java/duke/util/TaskList.java

```diff
@@ -154,12 +154,12 @@ public void printList(String type, String filterCondition) {
      * @throws InvalidIndexException if index given does not exist in the TaskList.
      */
     public String[] markTaskAsDone(int index) throws InvalidIndexException {
-        if (index <= 0 || index > tasksList.size()) {
-            throw new InvalidIndexException(tasksList.size());
+        if (index <= 0 || index > tasks.size()) {
+            throw new InvalidIndexException(tasks.size());
         }
         String[] message = {
                 "You completed a task! Maybe you aren't so incompetent after all.",
-                tasksList.get(index - 1).markTaskAsDone()
+                tasks.get(index - 1).markTaskAsDone()
```

Comment:
perhaps a String.format suffices? or changing the variable name to plural so it does not violate the naming standard?

Reply:
I have already changed the variable name here already but you're right, I definitely can use String.format to make this portion and similar parts more readable. 😃